src/service/iot_config.proto

```diff
@@ -348,6 +348,14 @@ message load_region_req_v1 {
 
 message load_region_res_v1 {}
 
+message gateway_location_req_v1 {
+  bytes gateway = 1;
+  uint64 timestamp = 2;
+  bytes signature = 3;
+}
+
+message gateway_location_res_v1 { string location = 1; }
+
```

Comment:
Should the config service  sign the response ? The caller should already have the signing pubkey

Reply:
We don't in any other message

Reply:
as i recall we had the config service sign responses to the gateways b/c we couldn't guarantee they were able to travel over a tls connection but in the case of the other network (router, hpr, etc) infrastructure we should be able to enforce a tls connection to the config service so the signature wasn't critical